examples/README

```diff
@@ -2,11 +2,11 @@ This directory contains examples of client code that uses kafka.
 
 To run the demo:
 
-   1. Start Zookeeper and the Kafka server
+   1. In Zookeeper mode, Start Zookeeper and the Kafka server. In KRaft mode, start the Kafka server. Note, it's unnecessary to start Zookeeper in KRaft mode.
```

Comment:
Thanks for the update @liuzhuang2017 , It's a little weird to mention that "it's unnecessary to start Zookeeper in KRaft mode" here since we already wrote how to operate in Zk mode. I think it's better to remove it.